{"name": "Test for change name", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.StaleElementReferenceException: Message: stale element reference: stale element not found in the current frame\n  (Session info: chrome=137.0.7151.68); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#stale-element-reference-exception\nStacktrace:\n#0 0xaaaadc14e934 <unknown>\n#1 0xaaaadbc464f4 <unknown>\n#2 0xaaaadbc4cad0 <unknown>\n#3 0xaaaadbc4e548 <unknown>\n#4 0xaaaadbc4e60c <unknown>\n#5 0xaaaadbc82de0 <unknown>\n#6 0xaaaadbc7eb28 <unknown>\n#7 0xaaaadbcc7b14 <unknown>\n#8 0xaaaadbc7d1cc <unknown>\n#9 0xaaaadc11b2a0 <unknown>\n#10 0xaaaadc11e2a8 <unknown>\n#11 0xaaaadc11de98 <unknown>\n#12 0xaaaadc106250 <unknown>\n#13 0xaaaadc11e8ec <unknown>\n#14 0xaaaadc0f00ac <unknown>\n#15 0xaaaadc13cd80 <unknown>\n#16 0xaaaadc13cf64 <unknown>\n#17 0xaaaadc14da8c <unknown>\n#18 0xffffb6ccee9c <unknown>\n#19 0xffffb6d37b18 <unknown>", "trace": "self = <test_exe.TestExe object at 0xffff8e6d0ed0>\n\n    @allure.title(\"Test for change name\")\n    def test_create_marker_and_change_name_marker(self):\n        self.login_page().open()\n        self.login_page().login()\n        self.menu().settings.open_setting_market()\n        self.markers_pages().open_create()\n        self.markers_pages().create.choose_enable()\n        self.markers_pages().create.fill_name(\"berlin\")\n        self.markers_pages().create.fill_configuration(\"babu\")\n        self.markers_pages().create.fill_market_type(\"Fortex\")\n        self.markers_pages().create.choose_save_to_csv_file()\n        self.markers_pages().create.choose_save_to_log_file()\n        self.markers_pages().create.click_create()\n        self.markers_pages().refresh_page()\n        self.markers_pages().find_row_by_username(\"berlin\")\n        self.markers_pages().click_edit_row_by_username(\"berlin\")\n>       self.markers_pages().edit.fill_market_type(\"OneZero\")\n\ntests/test_exe.py:29: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \npages/markers_page/components/edit.py:45: in fill_market_type\n    if market_type in element.text:\n../local/lib/python3.11/site-packages/selenium/webdriver/remote/webelement.py:109: in text\n    return self._execute(Command.GET_ELEMENT_TEXT)[\"value\"]\n../local/lib/python3.11/site-packages/selenium/webdriver/remote/webelement.py:572: in _execute\n    return self._parent.execute(command, params)\n../local/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py:448: in execute\n    self.error_handler.check_response(response)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.remote.errorhandler.ErrorHandler object at 0xffff8e639590>\nresponse = {'status': 404, 'value': '{\"value\":{\"error\":\"stale element reference\",\"message\":\"stale element reference: stale elemen...\\\\n#17 0xaaaadc14da8c \\\\u003Cunknown>\\\\n#18 0xffffb6ccee9c \\\\u003Cunknown>\\\\n#19 0xffffb6d37b18 \\\\u003Cunknown>\\\\n\"}}'}\n\n    def check_response(self, response: Dict[str, Any]) -> None:\n        \"\"\"Checks that a JSON response from the WebDriver does not have an\n        error.\n    \n        :Args:\n         - response - The JSON response from the WebDriver server as a dictionary\n           object.\n    \n        :Raises: If the response contains an error message.\n        \"\"\"\n        status = response.get(\"status\", None)\n        if not status or status == ErrorCode.SUCCESS:\n            return\n        value = None\n        message = response.get(\"message\", \"\")\n        screen: str = response.get(\"screen\", \"\")\n        stacktrace = None\n        if isinstance(status, int):\n            value_json = response.get(\"value\", None)\n            if value_json and isinstance(value_json, str):\n                import json\n    \n                try:\n                    value = json.loads(value_json)\n                    if len(value) == 1:\n                        value = value[\"value\"]\n                    status = value.get(\"error\", None)\n                    if not status:\n                        status = value.get(\"status\", ErrorCode.UNKNOWN_ERROR)\n                        message = value.get(\"value\") or value.get(\"message\")\n                        if not isinstance(message, str):\n                            value = message\n                            message = message.get(\"message\")\n                    else:\n                        message = value.get(\"message\", None)\n                except ValueError:\n                    pass\n    \n        exception_class: Type[WebDriverException]\n        e = ErrorCode()\n        error_codes = [item for item in dir(e) if not item.startswith(\"__\")]\n        for error_code in error_codes:\n            error_info = getattr(ErrorCode, error_code)\n            if isinstance(error_info, list) and status in error_info:\n                exception_class = getattr(ExceptionMapping, error_code, WebDriverException)\n                break\n        else:\n            exception_class = WebDriverException\n    \n        if not value:\n            value = response[\"value\"]\n        if isinstance(value, str):\n            raise exception_class(value)\n        if message == \"\" and \"message\" in value:\n            message = value[\"message\"]\n    \n        screen = None  # type: ignore[assignment]\n        if \"screen\" in value:\n            screen = value[\"screen\"]\n    \n        stacktrace = None\n        st_value = value.get(\"stackTrace\") or value.get(\"stacktrace\")\n        if st_value:\n            if isinstance(st_value, str):\n                stacktrace = st_value.split(\"\\n\")\n            else:\n                stacktrace = []\n                try:\n                    for frame in st_value:\n                        line = frame.get(\"lineNumber\", \"\")\n                        file = frame.get(\"fileName\", \"<anonymous>\")\n                        if line:\n                            file = f\"{file}:{line}\"\n                        meth = frame.get(\"methodName\", \"<anonymous>\")\n                        if \"className\" in frame:\n                            meth = f\"{frame['className']}.{meth}\"\n                        msg = \"    at %s (%s)\"\n                        msg = msg % (meth, file)\n                        stacktrace.append(msg)\n                except TypeError:\n                    pass\n        if exception_class == UnexpectedAlertPresentException:\n            alert_text = None\n            if \"data\" in value:\n                alert_text = value[\"data\"].get(\"text\")\n            elif \"alert\" in value:\n                alert_text = value[\"alert\"].get(\"text\")\n            raise exception_class(message, screen, stacktrace, alert_text)  # type: ignore[call-arg]  # mypy is not smart enough here\n>       raise exception_class(message, screen, stacktrace)\nE       selenium.common.exceptions.StaleElementReferenceException: Message: stale element reference: stale element not found in the current frame\nE         (Session info: chrome=137.0.7151.68); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#stale-element-reference-exception\nE       Stacktrace:\nE       #0 0xaaaadc14e934 <unknown>\nE       #1 0xaaaadbc464f4 <unknown>\nE       #2 0xaaaadbc4cad0 <unknown>\nE       #3 0xaaaadbc4e548 <unknown>\nE       #4 0xaaaadbc4e60c <unknown>\nE       #5 0xaaaadbc82de0 <unknown>\nE       #6 0xaaaadbc7eb28 <unknown>\nE       #7 0xaaaadbcc7b14 <unknown>\nE       #8 0xaaaadbc7d1cc <unknown>\nE       #9 0xaaaadc11b2a0 <unknown>\nE       #10 0xaaaadc11e2a8 <unknown>\nE       #11 0xaaaadc11de98 <unknown>\nE       #12 0xaaaadc106250 <unknown>\nE       #13 0xaaaadc11e8ec <unknown>\nE       #14 0xaaaadc0f00ac <unknown>\nE       #15 0xaaaadc13cd80 <unknown>\nE       #16 0xaaaadc13cf64 <unknown>\nE       #17 0xaaaadc14da8c <unknown>\nE       #18 0xffffb6ccee9c <unknown>\nE       #19 0xffffb6d37b18 <unknown>\n\n../local/lib/python3.11/site-packages/selenium/webdriver/remote/errorhandler.py:232: StaleElementReferenceException"}, "attachments": [{"name": 1749598512.6206567, "source": "e1a9a288-ad3e-4c2b-9333-b6dbe9da7c52-attachment.png", "type": "image/png"}], "start": 1749598524643, "stop": 1749598535354, "uuid": "9dcac5cb-0158-4344-9d77-9bed99c23128", "historyId": "f8de49c9b1fe105f7b7d48335c053509", "testCaseId": "f8de49c9b1fe105f7b7d48335c053509", "fullName": "tests.test_exe.TestExe#test_create_marker_and_change_name_marker", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_exe"}, {"name": "subSuite", "value": "TestExe"}, {"name": "host", "value": "3fcc72dcd74c"}, {"name": "thread", "value": "9-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_exe"}]}